lib/util/spmconfig.js

```diff
@@ -14,6 +14,21 @@ var unflatten = require('flat').unflatten
 var RC = require('rc-yaml-2')
 var fs = require('fs')
 var os = require('os')
+
+function setReceiverUrlsForRegion (region) {
+  if (region === 'EU' || region === 'eu') {
```

Comment:
If you do something like region = region.uppercase() or whatever the function, then you can make conditions a little simpler...

Reply:
And additional we would need a check if `region` exists, and check that `region` is a `string` and then we can call toUpperCase(). This would make the statement longer.